empress/support_files/js/colorer.js

```diff
@@ -553,6 +564,8 @@ define(["chroma", "underscore", "util"], function (chroma, _, util) {
         "#808000",
         "#008080",
     ];
+    Colorer.__PR_OR_GR = "PrOrGn";
+    Colorer.__prOrGr = ["green", "#fed8b1", "purple"];
```

Comment:
```suggestion
    Colorer.__prOrGr = ["#008000", "#fed8b1", "#800080"];
```
It's probably irrational on my part, but using hex codes just seems somehow "safer" to me -- in case Chroma.js would decide randomly that the default `green` should be another color, or whatever. (I doubt that'd ever happen, but IDK.)

Another motivation for using hex codes instead of color names is that it is consistent with how `Classic QIIME Colors` is declared above -- this could make this color map more portable to other projects that don't use Chroma.js.

Reply:
I'll add the hex codes. I doubt chroma.js would change the default colors but agree that we shouldn't rely on chroma.js to maintain its default colors.